metadata_backend/server.py

```diff
@@ -78,6 +78,7 @@ async def init() -> web.Application:
         web.delete("/users/{userId}", rest_handler.delete_user),
         web.post("/submit", submission_handler.submit),
         web.post("/validate", submission_handler.validate),
+        web.get("/mirror/{datasetId}", rest_handler.mirror_dataset),
```

Comment:
this endpoint we will need to put behind auth.

Reply:
done